app/src/main/java/com/gholem/moneylab/main/MainActivity.kt

```diff
@@ -62,8 +59,9 @@ class MainActivity : AppCompatActivity() {
 
     private fun showBottomNavigation(isVisible: Boolean) {
         binding.bottomNavigationFab.isEnabled = false
-        binding.bottomAppBar.isVisible = isVisible
-        binding.bottomNavigationFab.isVisible = isVisible
+        binding.bottomAppBar.isGone = !isVisible
```

Comment:
Why you switched to `isGone`? `isVisible` is better as you don't need to make a negation to match your logic, so change this one and others to:

binding.bottomAppBar.isVisible= isVisible